testdata/statement.ivy

```diff
@@ -159,3 +159,15 @@ x = 4 4 rho 0; y = 4 4; x[y[1]; y[2]] = 3; x
 	0 0 0 0
 	0 0 0 0
 	0 0 0 3
+
+# Order of evaluation
```

Comment:
i was going to add this to my fix for #95, but left it for you. i did need to fix one of the existing examples, demonstrating that the fix matters.